HEN_HOUSE/pegs4/density_corrections/compounds/cellulose.density

```diff
@@ -0,0 +1,32 @@
+  cellulose
```

Comment:
Can you include a short reference? I find that in that with time this is often useful: knowing where the author got the data, or how it was derived. Same comment for the silicon dioxide file

Reply:
how can  get efficiencies  for this detectors in specific energy ??